### ISSUE TITLE
Initialization of rmsprop optimizer in tensorflow

### ISSUE THREAD

**Issue by kvin97** [tensorflow/tensorflow#40670]
In tensorflow RMSPROP optimizer can be used with tf.compat.v1.train.RMSPropOptimizer function as follows.

tf.compat.v1.train.RMSPropOptimizer(
    learning_rate, decay=0.9, momentum=0.0, epsilon=1e-10, use_locking=False,
    centered=False, name='RMSProp'
)
But it doesn't convey how to initialize moving average of the squared gradient for each weight.  Can anyone tell me how does tensorflow initialize it?
